icu4c/source/test/intltest/restsnew.cpp

```diff
@@ -1395,6 +1396,15 @@ void NewResourceBundleTest::TestFilter() {
     }
 }
 
+/*
+ * The following test for ICU-20706 causes infinte loops on certain inputs for
+ * locales and calendars.  In order to unblock the build (ICU-21055), the test
+ * is comment out.
+ * The issue of the infinite loops and its blocking dependencies were captured
+ * in ICU-21080.
+ */
+#if 0
```

Comment:
Please limit the `#if 0 ... #endif` to just the two blocks of test values that "will cause infinity loop". We want to keep testing the ones that do terminate.

Reply:
Oh, right, of course.  Done.